Deepgram/Clients/Live/v1/Client.cs

```diff
@@ -663,110 +671,110 @@
             }
 
             Log.Debug("ProcessDataReceived", "Succeeded");
-            Log.Verbose("ProcessDataReceived", "LEAVE");
-            return;
+            Log.Verbose("LiveClient.ProcessDataReceived", "LEAVE");
         }
         catch (JsonException ex)
         {
-            Log.Error("ProcessDataReceived", $"JsonException: {ex}");
+            Log.Error("ProcessDataReceived", $"{ex.GetType()} thrown {ex.Message}");
+            Log.Verbose("ProcessDataReceived", $"Excepton: {ex}");
+            Log.Verbose("LiveClient.ProcessDataReceived", "LEAVE");
         }
         catch (Exception ex)
         {
-            Log.Error("ProcessDataReceived", $"Excepton: {ex}");
+            Log.Error("ProcessDataReceived", $"{ex.GetType()} thrown {ex.Message}");
+            Log.Verbose("ProcessDataReceived", $"Excepton: {ex}");
+            Log.Verbose("LiveClient.ProcessDataReceived", "LEAVE");
         }
-
-        Log.Verbose("ProcessDataReceived", "LEAVE");
     }
 
     /// <summary>
     /// Closes the Web Socket connection to the Deepgram API
     /// </summary>
     /// <returns>The task object representing the asynchronous operation.</returns>
-    public async Task Stop(CancellationTokenSource? cancellationToken = null)
+    public async Task Stop(CancellationTokenSource? cancelToken = null)
     {
-        Log.Verbose("Stop", "ENTER");
-
-        var cancelToken = _cancellationTokenSource.Token;
-        if (cancellationToken != null)
-        {
-            Log.Verbose("Stop", "Using provided cancellation token");
-            cancelToken = cancellationToken.Token;
-        } 
+        Log.Verbose("LiveClient.Stop", "ENTER");
 
         // client is already disposed
         if (_clientWebSocket == null)
         {
-            Log.Warning("Stop", "Client has already been disposed");
-            Log.Verbose("Stop", "LEAVE");
+            Log.Information("Stop", "Client has already been disposed");
+            Log.Verbose("LiveClient.Stop", "LEAVE");
             return;
         }
 
-        // send the close message and flush transcription message
-        try {
+        if (cancelToken == null)
+        {
+            Log.Information("Stop", "Using default disconnect cancellation token");
+            cancelToken = new CancellationTokenSource(Constants.DefaultConnectTimeout);
+        }

```

Comment:
Will update!

Reply:
Actually, this isn't needed. We need to create the cancel token for `Stop()` because this handles any calls to functions within `Stop()` that might actually need to be called should the socket be open. Ignoring suggestion.